src/app/static/src/app/store/modelRun/mutations.ts

```diff
@@ -41,6 +41,7 @@ export const mutations: MutationTree<ModelRunState> = {
 
     [ModelRunMutation.RunResultFetched](state: ModelRunState, action: PayloadWithType<ModelResultResponse>) {
         state.result = action.payload;
+        state.warnings = action.payload.warnings
```

Comment:
Note this mutation is invoked twice - [once during fitting](https://github.com/mrc-ide/hint/blob/master/src/app/static/src/app/store/modelRun/actions.ts#L48) and [once during calibration](https://github.com/mrc-ide/hint/blob/master/src/app/static/src/app/store/modelCalibrate/actions.ts#L74)